Detectors/TOF/base/src/Geo.cxx

```diff
@@ -0,0 +1,20 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "TOFBase/Geo.h"
+
+ClassImp(o2::tof::Geo)
+
+  using namespace o2::tof;
```

Comment:
wrong indentation due to a clang-format bug. This can be solved by putting `;` or even better just put ClassImp in the last line.